doT.js

```diff
@@ -24,7 +24,8 @@
 			doNotSkipEncoded: false
 		},
 		template: undefined, //fn, compile template
-		compile:  undefined  //fn, for express
+		compile:  undefined, //fn, for express
+		log: false
```

Comment:
@halton it should be true by default, as otherwise this change is not backward compatible.